concepticondata/conceptlists/BirketSmith-1928-510.tsv

```diff
@@ -0,0 +1,511 @@
+BirketSmith-1928-510-349	349	Father’s brother	1985	PATERNAL UNCLE (FATHER'S BROTHER)	Relationship
+BirketSmith-1928-510-350	350	Father’s sister	2691	PATERNAL AUNT (FATHER'S SISTER)	Relationship
+BirketSmith-1928-510-351	351	Mother’s brother	2692	MATERNAL UNCLE (MOTHER'S BROTHER)	Relationship
+BirketSmith-1928-510-352	352	Mother’s sister	2693	MATERNAL AUNT (MOTHER'S SISTER)	Relationship
+BirketSmith-1928-510-353	353	Cousin (male line)			Relationship
+BirketSmith-1928-510-354	354	Cousin (female line)			Relationship
+BirketSmith-1928-510-355	355	Man’s brother’s child			Relationship
+BirketSmith-1928-510-356	356	Woman’s brother’s child			Relationship
+BirketSmith-1928-510-357	357	Man’s sister’s child			Relationship
+BirketSmith-1928-510-358	358	Woman’s sister’s child			Relationship
+BirketSmith-1928-510-359	359	Father in law	1055	FATHER-IN-LAW	Relationship
+BirketSmith-1928-510-360	360	Mother in law	1050	MOTHER-IN-LAW	Relationship
+BirketSmith-1928-510-361	361	Son in law	1056	SON-IN-LAW	Relationship
+BirketSmith-1928-510-362	362	Man’s sister’s husband			Relationship
+BirketSmith-1928-510-363	363	Daughter in law	1057	DAUGHTER-IN-LAW	Relationship
+BirketSmith-1928-510-364	364	Woman’s brother’s wife			Relationship
+BirketSmith-1928-510-365	365	Wife’s brother			Relationship
+BirketSmith-1928-510-366	366	Husband’s sister			Relationship
+BirketSmith-1928-510-367	367	Man’s brother’s wife			Relationship
+BirketSmith-1928-510-368	368	Wife’s sister			Relationship
+BirketSmith-1928-510-369	369	Woman’s sister’s husband			Relationship
+BirketSmith-1928-510-370	370	Husband’s brother			Relationship
+BirketSmith-1928-510-371	371	Wife’s younger sister’s husband			Relationship
+BirketSmith-1928-510-372	372	Husband’s younger brother’s wife			Relationship
+BirketSmith-1928-510-373	373	Wife’s elder sister’s husband			Relationship
+BirketSmith-1928-510-374	374	Husband’s elder brother’s wife			Relationship
+BirketSmith-1928-510-375	375	Son’s father-in-law			Relationship
+BirketSmith-1928-510-376	376	Daughter’s father-in-law			Relationship
+BirketSmith-1928-510-377	377	Son’s mother-in-law			Relationship
+BirketSmith-1928-510-378	378	Daughter’s mother-in-law			Relationship
+BirketSmith-1928-510-379	379	Adopted father			Relationship
+BirketSmith-1928-510-380	380	Adopted mother			Relationship
+BirketSmith-1928-510-381	381	Adopted son			Relationship
+BirketSmith-1928-510-382	382	Adopted daughter			Relationship
+BirketSmith-1928-510-383	383	I	1209	I	Pronouns
+BirketSmith-1928-510-384	384	You (sing.)	1215	THOU	Pronouns
+BirketSmith-1928-510-385	385	He, she it	262	HE OR SHE OR IT	Pronouns
+BirketSmith-1928-510-386	386	We (dual.)	2634	WE TWO	Pronouns
+BirketSmith-1928-510-387	387	We (plur.)	1212	WE	Pronouns
+BirketSmith-1928-510-388	388	You (dual.)	2111	YOU TWO	Pronouns
+BirketSmith-1928-510-389	389	You (plur.)	1213	YOU	Pronouns
+BirketSmith-1928-510-390	390	They (dual. & plur.)	817	THEY	Pronouns
```

Comment:
I don't know what the original list looks like but would it allow splitting this concept as well? (Same goes for BirketSmith-1928-510-392)

Reply:
I'll have a look, thanks.

Reply:
Both are cases of colexifications so I'd say the glosses can be left as is.